.github/workflows/test_ci.yml

```diff
@@ -34,4 +34,5 @@ jobs:
       - name: Build
         run: npm run build
       - name: Integration Test
+        # run build before integration as we are static serving build file for integration testing
```

Comment:
Dont really understand this comment here.

Reply:
Previously we used npm run start to serve files, however, testcafe proxy (hammerhead) is blocking a call made to ethereum-ens from opengsn causing integration test to fail, hence we used a statically built copy served on a http server instead to run testcafe test which requires files to be built first.